server/src/http/routes/formationsRoutes.js

```diff
@@ -78,5 +80,32 @@ module.exports = () => {
     })
   );
 
+  router.get(
+    "/api/inserjeunes/formations/uai/:uai/code_formation/:code_formation/millesime/:millesime",
+    tryCatch(async (req, res) => {
+      const { uai, code_formation, millesime } = await validate(
+        { ...req.params, ...req.query },
+        {
+          uai: Joi.string()
+            .pattern(/^[0-9]{7}[A-Z]{1}$/)
+            .required(),
+          code_formation: Joi.string().required(),
+          millesime: Joi.string().required(),
+        }
+      );
+
+      let found = await formationsStats().findOne(
+        { uai, code_formation, millesime },
+        { projection: { _id: 0, _meta: 0 } }
+      );
+
+      if (!found) {
+        throw Boom.notFound("UAI, code formation et/ou millésime invalide");
+      }
+
+      res.json(found);
```

Comment:
Je sais que ce n'est pas indispensable mais j'aime bien le return pour m'assurer qu'il n'y aura pas de code exécuté ensuite :
```suggestion
      return res.json(found);
```

Reply:
Oui exact moi aussi